sdk/nodejs/policy/tests/server.spec.ts

```diff
@@ -901,6 +902,57 @@ describe("makeRemediateRpcFun", () => {
             assert.equal(response!.getNotApplicableList()[0].getReason(), "Policy does not implement remediateResource");
         }));
     });
+
+    describe("ReportViolation with object argument", () => {
```

Comment:
I think this should be within the top-level `describe("makeAnalyzeRpcFun", () => {`, so right after line 476.